### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by zmraul**
Addresses [DPE-1010](https://warthogs.atlassian.net/browse/DPE-1010).

Change docs to follow current readme scheme. 

[DPE-1010]: https://warthogs.atlassian.net/browse/DPE-1010?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,128 @@
...
+# generate shared internal key
+openssl genrsa -out internal-key.pem 3072
+# generate external keys for each unit
+openssl genrsa -out external-key-0.pem 3072
+openssl genrsa -out external-key-1.pem 3072
+openssl genrsa -out external-key-2.pem 3072
+# apply both private keys on each unit, shared internal key will be allied only on juju leader
+juju run-action kafka-k8s/0 set-tls-private-key "external-key=$(base64 -w0 external-key-0.pem)"  "internal-key=$(base64 -w0 internal-key.pem)"  --wait
+juju run-action kafka-k8s/1 set-tls-private-key "external-key=$(base64 -w0 external-key-1.pem)"  "internal-key=$(base64 -w0 internal-key.pem)"  --wait
+juju run-action kafka-k8s/2 set-tls-private-key "external-key=$(base64 -w0 external-key-2.pem)"  "internal-key=$(base64 -w0 internal-key.pem)"  --wait
```
Comment (deusebio) [inline README.md:106]: I'm not sure we have the `external-key` option in that action. I believe we have to adapt the Config section as well

**Review by deusebio — CHANGES_REQUESTED** [review]
Great work @zmraul !! Looks that this docs is really shaping up!!!!

Just few points before approving:

1. As pointed out in the comments, there are some misalingment between documentation and charm features (just make sure that the commands we put here they do work with a dry run)
2. I believe we need to give some love to the metadata too. Description and summary are missing there, and make sure we put the right info for the other fields
3. I believe it would be good to also put some texts about storage handling in the usage as we have this feature in (marc can maybe help if you are unsure)
4. I believe it would be good to also put some texts about the data-integrator (paolo can maybe help if you are unsure)

File: README.md [hunk 2]
```diff
@@ -1,43 +1,169 @@
...
-- [ ] Rack awareness support
-- [ ] Persistent storage support with [Juju Storage](https://juju.is/docs/olm/defining-and-using-persistent-storage)
-- [ ] TLS/SSL support
+This is used for logs storage, mounted on `/logs/kafka`
+
+## Relations
+
+Supported [relations](https://juju.is/docs/olm/relations):
+
+#### `data-integrator` interface:
```
Comment (welpaolo) [inline README.md:79]: I think the interface name should be kafka-client. This interface is implemented in the data-integrator.
Suggestion: maybe you can copy/paste two sentences to add a description for the data-integrator

**Review by welpaolo — APPROVED** [review]
Great improvement! I left a small comment

**Review by deusebio — APPROVED** [review]
Looks good!

**Review by welpaolo — APPROVED** [review]
Great work!

**Pull request merged**